ui/src/components/Canvas.tsx

```diff
@@ -221,30 +240,25 @@ function ResultBlock({ pod, id }) {
           )}
         </Box>
       )}
-      {pod.stdout && (
-        <Box overflow="scroll" border="1px">
-          {/* TODO separate stdout and stderr */}
-          <Box bgcolor="lightgray">Stdout</Box>
+
+      {pod.running && <CircularProgress />}
+      <Box overflow="scroll" maxHeight="145px" border="1px">
+        {/* <Box bgcolor="lightgray">Error</Box> */}
+        {pod.stdout && (
           <Box whiteSpace="pre-wrap" fontSize="sm">
```

Comment:
FYI this `fontSize="sm"` was a bug in my previous code; I was using another library other than MUI, and didn't update this code. The MUI way to set size is something like this, using [sx](https://mui.com/system/getting-started/the-sx-prop/) property:

```
<Box sx={{fontSize: 10}}></Box>
```